content/terraform/v1.12.x/docs/language/block/resource.mdx

```diff
@@ -82,13 +82,13 @@ The following `resource` block defines all of the supported built-in arguments y
 ```hcl
 resource "<TYPE>" "<LABEL>" {
   <PROVIDER_ARGUMENTS>
-  count = <NUMBER>      # `for_each` and `count` are mutually exclusive 
+  count = <NUMBER>      # `for_each` and `count` are mutually exclusive
```

Comment:
Oh my gosh my dang linter is vicious, there is only one link change in this whole file, everything else is whitespace, sorry!